README.md

```diff
@@ -187,8 +184,8 @@ end
 
 ### Usage ###
 
-form = CompanyRegistrationForm.new(params[:company])
-if form.submit
+form = CompanyRegistrationForm.call(params[:company])
```

Comment:
Don't fully clear what attributes required in `.call`. 
Need to inform that it can be 
- hash: `.call({ name:  'test', count: 5 })`
- named attributes: .call(name: 'test', count: 5)`

Reply:
Forms are to mimic the model's behavior. The same case you have when you do something like `User.new` or `User.create`.